internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/integration/utils/VeniceControllerWrapper.java

```diff
@@ -114,7 +115,8 @@ private VeniceControllerWrapper(
       boolean isParent,
       List<ServiceDiscoveryAnnouncer> d2ServerList,
       String zkAddress,
-      MetricsRepository metricsRepository) {
+      MetricsRepository metricsRepository,
+      String coloName) {
```

Comment:
nit: Can we make this the first arg of the function? Similar for other ComponentWrappers

Reply:
Yep, this can be done